src/flownet/config_parser/_config_parser.py

```diff
@@ -35,9 +71,11 @@ def create_schema(_to_abs_path) -> Dict:
                                     "input_case": {
                                         MK.Type: types.String,
                                         MK.Transformation: _to_abs_path,
+                                        MK.Description: "Eclipse input case to be used as data source for FlowNet",
```

Comment:
```suggestion
                                        MK.Description: "Simulation input case to be used as data source for FlowNet",
```

Reply:
We can do that, the argument for writing Eclipse is that currently we sort of require Eclipse input case (Flow does not support writing `WSTATUS` which is used to instruct well status in FlowNet). 

Still change to `Simulation` @wouterjdb?

Reply:
Yeah, let's just change it.